server/src/main/java/org/opensearch/action/admin/cluster/shards/routing/weighted/put/ClusterPutWeightedRoutingResponse.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.action.admin.cluster.shards.routing.weighted.put;
+
+import org.opensearch.action.support.master.AcknowledgedResponse;
+import org.opensearch.common.io.stream.StreamInput;
+import org.opensearch.common.xcontent.ToXContentObject;
+
+import java.io.IOException;
+
+/**
+ * Response from updating weights for weighted round-robin search routing policy.
+ *
+ * @opensearch.internal
+ */
+public class ClusterPutWeightedRoutingResponse extends AcknowledgedResponse implements ToXContentObject {
+    public ClusterPutWeightedRoutingResponse(boolean acknowledged) {
+        super(acknowledged);
+    }
+
+    public ClusterPutWeightedRoutingResponse(StreamInput in) throws IOException {
+        super(in);
+    }
+}
```

Comment:
Whats the point extending?

Reply:
Not needed to implement ```ToXContentObject``` since ```AcknowledgedResponse``` takes care of serialization/deserialization